views/tables/ChartsTable/ChartsTable.test.jsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import moment from 'moment'
+
+import ChartsTable, { DATE_FORMAT } from '.'
+import { createChart, createUser } from '../../../test/fixtures'
+import { render, screen, within } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { routes } from '../../routes/routes'
+
+describe(ChartsTable, () => {
+  const user = createUser({ uid: 'user' })
+  const charts = [
+    createChart({
+      title: 'first chart',
+      updatedAt: '2023-06-28T15:43:23.789Z',
```

Comment:
what happens if we have a chart without this property

Reply:
is that possible? I thought all charts had this.

Reply:
This property is missing and we it looks like there's an opportunity here to capture it

Reply:
we're already handling this property not being present, so I think we're covered 👍